data-reconciliation-app/demo/scripts/01-data_ingest_demo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+sudo rm -rf ./demo/Reports/*.json
+VARS_PATH="./demo/scripts/env_vars.sh"
+. "$VARS_PATH"
+
+cd "${certs}"
+
+printf "Member1 trying to ingest data \n" 
+check_eq "Member1 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member0) -H "Content-Type: application/json" --data-binary "@../../demo/data/member1_demo_pt1.json" $only_status_code)"
+addCheckpoint "✅ Member1 successfully ingested data"
+
+# printf "Member2 trying to ingest empty data \n"
+check_eq "Member2 - data ingest failed (data is null or empty)" "400" "$(curl $ingestUrl -X POST $(cert_arg member1) -H "Content-Type: application/json" --data-binary "" $only_status_code)"
+addCheckpoint "✅ Member2 correctly failed to ingest empty data"
```

Comment:
nit:

`member correctly failed.` seems a little weird to me.
I would go with `Application successfully blocked ingesting empty data`

Reply:
@ross-p-smith, @kusweta  , this clashes with a previous review that has been already addressed. So feel free to comment on it and discard if you disagree.

Reply:
I think green tick goes better with the comment you suggested @julioalex-rezende